bin/vtshave.js

```diff
@@ -86,15 +72,9 @@ if (argv.maxzoom) opts.maxzoom = argv.maxzoom;
 shaver.shave(buffer, opts, function(err, shavedBuffer) {
     if (err) throw err.message;
 
-    if (is_compressed) {
-      var decompressed_og = zlib.gunzipSync(buffer);
-      console.log('Before:\n',JSON.stringify(vtinfo(decompressed_og),null,1));
-      var decompressed_res =   zlib.gunzipSync(shavedBuffer);
-      console.log('Before:\n',JSON.stringify(vtinfo(decompressed_res),null,1));
-    } else {
-      console.log('Before:\n',JSON.stringify(vtinfo(buffer),null,1));
-      console.log('After:\n',JSON.stringify(vtinfo(shavedBuffer),null,1));
-    }
+    console.log('Before:\n',bytes(buffer.length));
+    console.log('After:\n',bytes(shavedBuffer.length));
+    console.log('Savings:\n',(shavedBuffer.length/buffer.length*100).toFixed(2)+'%');
```

Comment:
This change is a little outside the original purpose of this PR, but nevertheless a good change to facilitate this PR. The request was made to list `vtshave.js` as a public CLI. This change avoids needing to force more dependencies on downstream users. It also, in my opinion, improves the UX of the tool. I don't think this too should be trying to output complex diffs, that is better done by downstream tools that can read vtiles.